draft-ietf-httpbis-sfbis.md

```diff
@@ -966,16 +968,23 @@ Given an ASCII string as input_string, return a Date. input_string is modified t
 
 Given an ASCII string as input_string, return a string of Unicode characters. input_string is modified to remove the parsed value.
 
-0. If the first character of input_string is not "%", fail parsing.
-1. Discard the first character of input_string.
-2. Let parsed_string be the result of running Parsing a String ({{parse-string}}) with input_string.
-3. Let byte_array be the result of applying ASCII encoding to input_string.
-4. For each sigil_byte in byte_array which is %25 ("%"):
-   1. Let octet_hex be the two bytes after sigil_byte in byte_string. If there are not two bytes, fail parsing.
-   2. Let octet be the result of decoding octet_hex as hexidecimal, in a case-insensitive fashion.
-   3. Replace sigil_byte and octet_hex in byte_array with octet.
-5. Let unicode_string be the result of decoding byte_array as a UTF-8 string {{UTF8}}. Fail parsing if decoding fails.
-6. Return unicode_string.
+0. If the first two characters of input_string are not  %x25 ("%") followed by DQUOTE, fail parsing.
```

Comment:
Maybe define "PERCENT" upfront and use it throughout.

Reply:
I've changed to just using "%", to align with other usage.